model/__init__.py

```diff
@@ -430,12 +433,12 @@ def initialize_data(cls, session, set_site_configuration=True):
         # it was updated by cls.update_timestamps_table
         return session
 
-def production_session():
+def production_session(initialize_data=True):
```

Comment:
I don't know why this doesn't happen the _first_ time `DatabaseMigrationScript` calls production_session, but the second time it calls that method (because it screwed up the first database connection), it tries to initialize the dataset, which we don't want to do until the schema is in good shape.